tasks/build-fits-site.yml

```diff
@@ -0,0 +1,83 @@
+- name: Build site for FITS microservice
+  blockinfile:
+    path: "/etc/apache2/conf-available/CrayFits.conf"
+    create: yes
+    block: |
+      Alias "/crayfits" "/var/www/html/CrayFits/public"
```

Comment:
Should make `/var/www/html/` into a configurable variable so it can be overridden like https://github.com/Islandora-Devops/ansible-role-crayfish/blob/master/defaults/main.yml#L11